src/python/bot/startup/run_bot.py

```diff
@@ -158,6 +160,10 @@ def task_loop():
       task.dont_retry()
       exception_occurred = True
       stacktrace = traceback.format_exc()
+    except google.auth.exceptions.RefreshError:
```

Comment:
Is this not already caught by the existing catchall on line 167? the resulting behaviour seems like it would be the same?